ports/servoshell/desktop/app_state.rs

```diff
@@ -112,18 +112,30 @@ impl RunningAppState {
         }
     }
 
-    pub(crate) fn new_toplevel_webview(self: &Rc<Self>, url: Url) {
+    pub(crate) fn new_toplevel_webview(self: &Rc<Self>, url: Url) -> WebViewId {
+        self.new_toplevel_webview_with_focus(url, true)
+    }
```

Comment:
Let's remove the argument and just have two variants of this function that return `WebView`, which allows operating on the return value.

```suggestion
    pub(crate) fn create_and_focus_toplevel_webview(self: &Rc<Self>, url: Url) -> WebView {
        let webview = self.create_toplevel_webview(url, true);
        webview.focus();
        webview.raise_to_top(true);
    }
```